fe/fe-core/src/test/java/org/apache/doris/nereids/rules/analysis/CheckRowPolicyTest.java

```diff
@@ -137,8 +137,8 @@ public void checkOnePolicy() throws Exception {
         Assertions.assertTrue(plan instanceof LogicalFilter);
         LogicalFilter filter = (LogicalFilter) plan;
         Assertions.assertEquals(filter.child(), relation);
-        Assertions.assertTrue(filter.getPredicates() instanceof EqualTo);
-        Assertions.assertTrue(filter.getPredicates().toString().contains("k1 = 1"));
+        Assertions.assertTrue(filter.getConjuncts().get(0) instanceof EqualTo);
```

Comment:
all get(0) should be replaced by `filter.getPredicate()`, and `filter.getPredicate` should call `ExpressionUtils.and(getConjuncts())`, please check by urself